loginPage/tests.py

```diff
@@ -351,6 +351,69 @@ def test_failing_user_creation(self):
             email__iexact='test@example.com'
         ).delete()
 
+    def testLogout(self):
+        # Set up driver.
+        driver = self.driver
+        driver.get('http://127.0.0.1:8000/')
```

Comment:
I don't really like this hardcoded URL, but that's easy to change, and your tests don't use the LiveServerTestCase (yet) so I'm not sure it will necessarily matter... But it'd be good to dynamically retrieve the URL instead.